### ISSUE TITLE
Supporting JSON Schema spes 2019-09 and 2020-12

### ISSUE THREAD

**Issue by NoahTheDuke** [luposlip/json-schema#13]
Hey there! Thanks for this library, it's really helpful. Is there any interest in supporting later versions of the JSON Schema specification? There are some features I'd love to use that I can't currently. I'm willing to help if you're open to it.

**Comment by luposlip** [comment]
Thanks for reaching out.
I'm definitely open to supporting later standards, as long as it doesn't
mean heavier dependencies, slower validation or in other ways worse
experience dør current users 🙂

fre. 3. feb. 2023 16.30 skrev Noah ***@***.***>:

**Comment by NoahTheDuke** [comment]
I see that you're relying on [Everit JSON Schema](https://github.com/everit-org/json-schema), which hasn't updated past Draft 7. Would you be willing to change to a different Java JSON Schema validator library?

**Comment by luposlip** [comment]
Definitely, if it works as good (ie. nice validation messages) and is as
performant as the current one.

**Comment by luposlip** [comment]
Which Java library do you have in mind?

**Comment by luposlip** [comment]
Any updates @NoahTheDuke ?
